core/workspace.ts

```diff
@@ -689,7 +689,7 @@ export class Workspace implements IASTNodeLocation {
    *
    * @param func Function to stop calling.
    */
-  removeChangeListener(func: Function) {
+  removeChangeListener(func: (e: Abstract) => void) {
```

Comment:
This is a breaking change, though not I think one that would interfere with the method being used as intended.

Still: why not allow developers to pass in any function (or indeed any value)?

Reply:
Like you said, I don't think this would actually break anyone (but please correct me if I'm wrong). I think it's helpful to tell devs what values we expect.